<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatrixWidget</class>
 <widget class="QWidget" name="MatrixWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>120</width>
    <height>117</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="border_widget" native="true">
     <property name="styleSheet">
      <string notr="true">background-color: #eee;</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="name_label">
        <property name="styleSheet">
         <string notr="true">font: bold;</string>
        </property>
        <property name="text">
         <string>Matrix Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="size_label">
        <property name="text">
         <string>1000x1000</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="block_size_label">
        <property name="text">
         <string>1x1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="other_label">
        <property name="text">
         <string>Etc</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
